community/contributing/bisect_workflow.rst

```diff
@@ -0,0 +1,131 @@
+.. _doc_bisect_workflow:
+
+Bisect workflow
+===============
+
+.. highlight:: shell
+
+After reporting a bug on the
+`Godot repository on GitHub <https://github.com/godotengine/godot>`__,
+you may be asked by a contributor to *bisect* the issue. Bisecting makes
```

Comment:
I think the first sentence should mention regressions, since that's the main use case for bisecting.